packages/bus/src/utils/RandomString.ts

```diff
@@ -0,0 +1,11 @@
+export const RundomString = (length: number): string => {
```

Comment:
Remove unused function

Reply:
This function is used to generate consumerTag for testing.

Reply:
I see. Anyway, such functions that you use only to generate fixtures or test data don't belong here. You should place them inside spec files. In this particular case, I don't see any reason to use this function at all. The consume tag might be any string.